pybossa/api/project.py

```diff
@@ -149,7 +149,9 @@ def _validate_instance(self, project):
             msg = "Project short_name is not valid, as it's used by the system."
             raise ValueError(msg)
         ensure_user_assignment_to_project(project)
-        validate_ownership_id(project.info.get('ownership_id'))
+        if not valid_ownership_id(project.info.get('ownership_id')):
+            ownership_id_title = current_app.config.get('OWNERSHIP_ID_TITLE', 'Ownership ID')
+            raise ValueError(f"{ownership_id_title} must be numeric and less than 20 characters.")
```

Comment:
Seems inconsistent behavior with the error is getting reported via api above and from fronted [here](https://github.com/bloomberg/pybossa/pull/1055/files#diff-1eee30bb7558109986f503403bf5476f73a6252232dda03611d869f344e53b1dR259-R270) as the error messages are different; line above missing `Got :<input-number>`. It would be nice to have a single place from where a consistent error message is reported back both for api or frontend. This can be achieved via existing similar implementations [here](https://github.com/bloomberg/pybossa/blob/main/pybossa/repositories/project_repository.py#L70-L74)

3.